app/views/service_unavailable.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components._
+@import views.html.Layout
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcNewTabLink
+@import uk.gov.hmrc.hmrcfrontend.views.Aliases.NewTabLink
+
+@this(
+    layout: Layout,
+    p: components.p,
+    link: components.link,
+    h1: components.h1,
+    hmrcNewTabLink: HmrcNewTabLink
+)
+
+@(backlink: Option[String] = None)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
+
+@layout(pageTitle = Some(messages("cf.service-unavailable.title")) , backLink = backlink, deskpro = false) {
+
+@h1(msg = "cf.service-unavailable.heading", classes = "govuk-heading-xl  govuk-!-margin-bottom-6", id = Some("service-unavailable.heading"))
+
+@p(
+    id = Some("older-statement-guidance-text"),
+    classes = "govuk-body govuk-!-margin-bottom-6",
```

Comment:
not a biggie.. we can use this class `govuk-!-margin-bottom-7` just for consistency purpose :)

Reply:
govuk-!-margin-bottom-6 class has been removed after cross checking with Prototype and spacing looks fine now. Also shown the page (after the change) to Donald and he is happy with it. Hence resolving